operators/src/main/scala/ml/sparkling/graph/operators/algorithms/pregel/Pregel.scala

```diff
@@ -0,0 +1,50 @@
+package ml.sparkling.graph.operators.algorithms.pregel
+
+import org.apache.spark.graphx._
+
+import scala.reflect.ClassTag
+
+/**
+  * Created by mth on 4/19/17.
+  */
+object Pregel extends Serializable {
+
+  def apply[OVD, VD: ClassTag, ED, MD: ClassTag](graph: Graph[OVD, ED],
+                                                 prepareVertices: (OVD) => VD,
+                                                 vpred: (Int) => (VertexId, VD, Option[MD]) => VD,
+                                                 send: (Int) => (EdgeContext[VD, ED, MD]) => Unit,
+                                                 merge: (MD, MD) => MD,
+                                                 interruptAfterEmptyRounds: Int = 1) = {
+
+
+    var round = 0
+    var numOfEmptyRounds = 0
```

Comment:
counter of rounds with no messages (need in one of BC algorithm where rounds with no messages is likely)